README.md

```diff
@@ -34,13 +39,55 @@ TODO: use commands with parameters for examples
 
 ### HTTP
 
+Here is a sample to get list of table via HTTP protocol.
+
     Groonga::Client.open(:host => host, :port => port, :protocol => :http) do |client|
       tables = client.table_list
       tables.each do |table|
         table.name
       end
     end
 
+### Typical example
+
+Here is a typical example to learn usage of groonga-client. In this example, it creates `User` table and
+`login_name` column. Then it loads sample data and select a person which contains `bob` as a key.
+
+    Groonga::Client.open(:host => host, :port => port, :protocol => :http) do |client|
+      client.table_create(:name => :User,
+                          :flags => :TABLE_PAT_KEY,
+                          :key_type => :ShortText)
+      client.column_create(:table => :User,
+                           :name => :login_name,
+                           :flags => :COLUMN_SCALAR,
+                           :type => :ShortText)
+      values = [
+        {
+          :_key => "bob",
+          :login_name => "Bob"
+        },
+        {
+          :_key => "tim",
+          :login_name => "Tim"
+        },
+        {
+          :_key => "jessie",
+          :login_name => "Jessie"
+        },
+      ]
+      client.load(:table => :User,
+                  :values => values.to_json)
+      result = client.select(:table => :User, :query => '_key:bob')
+      if result.success?
+        puts result.n_hits
+        result.records.each do |record|
+          puts record['login_name']
```

Comment:
Use double quote instead of single quote. Normally, we shouldn't combine multiple styles in one code.